### PR TITLE
Feat: canVote integration

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

canVote uses now real contract call

Task: APP-1218

Depends on: #106 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder of the package after the [UPCOMING] title and before the latest version.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: modules/common/CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ TEMPLATE:
 - Fix 1, 2, 3
 -->
 
-## [Upcomfing]
+## [0.6.1-alpha]
```
Comment (brickpop) [inline modules/common/CHANGELOG.md]: Extremely low importance, but adding a feature should be a minor release, not a patch

File: modules/common/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-common",
   "author": "Aragon Association",
-  "version": "0.6.0-alpha",
+  "version": "0.6.1-alpha",
```
Comment (brickpop) [inline modules/common/package.json]:
```suggestion
  "version": "0.7.0-alpha",
```

File: modules/common/CHANGELOG.md (hunk 1 above)
Comment (brickpop) [reply modules/common/CHANGELOG.md]: Suggesting `0.7.0-alpha`

File: modules/client/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-client",
   "author": "Aragon Association",
-  "version": "0.12.0-alpha",
+  "version": "0.13.0-alpha",
   "license": "MIT",
   "main": "dist/index.js",
   "module": "dist/sdk-client.esm.js",
```
Comment (brickpop) [inline modules/client/package.json:7]: We should also be bumping the `client` dependency on `common`'s version

File: modules/client/test/integration/client-addressList/methods.test.ts [hunk 4]
```diff
@@ -184,7 +184,7 @@ describe("Client Address List", () => {
 
       const params: ICanVoteParams = {
         address: "0x1234567890123456789012345678901234567890",
-        proposalId: "0x1234567890123456789012345678901234567890_0x1",
+        proposalId: "0x1234567890123456789012345678901234567890",
```
Comment (josemarinas) [inline modules/client/test/integration/client-addressList/methods.test.ts:187]: I believe this should be a `BigInt` if we are not using the subgraph definition for proposalID

Comment (brickpop) [reply modules/client/test/integration/client-addressList/methods.test.ts:187]: mmm here I disagree
Identifiers that can be copied+pasted by the user should be hex strings. Low level binary values should be exchanged in a language-native way, but I don't think this is the case

**Review by brickpop — APPROVED** [review]

**Pull request merged**